packages/calypso-e2e/src/lib/pages/stats-page.ts

```diff
@@ -16,6 +16,13 @@ export class StatsPage {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
+		// Surppress notices.
```

Comment:
Suppressed traffic page tooltips

Reply:
**issue:blocking**- this does not actually suppress the notice, as the `evaluate` method needs is an async call. 

As it stands, this change is not likely to result in the problem being fixed.